features/telemetry/protocols/collection/src/main/java/org/opennms/netmgt/telemetry/protocols/collection/AbstractPersistingAdapter.java

```diff
@@ -1,8 +1,8 @@
 /*******************************************************************************
  * This file is part of OpenNMS(R).
  *
- * Copyright (C) 2017-2017 The OpenNMS Group, Inc.
- * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ * Copyright (C) 2019 The OpenNMS Group, Inc.
```

Comment:
If the file was created before and updated, the copyright headers should state `<createYear>-<currentYear>`, in this case `2017-2019`.